mvrx/src/main/kotlin/com/airbnb/mvrx/BaseMvRxFragment.kt

```diff
@@ -11,6 +11,14 @@ import androidx.lifecycle.LifecycleOwner
  */
 abstract class BaseMvRxFragment : Fragment(), MvRxView {
 
+<<<<<<< HEAD
```

Comment:
@mmcwong Looks like you accidentally left some merge conflicts in

Reply:
whoops!